lib/editor/selectors/timetable.js

```diff
@@ -170,7 +172,9 @@ const isCellValueInvalid = (
         previousIndex--
       }
       // Ensure value is increasing over previous value
-      const isInvalid = val < previousValue
+      // Flexible stop times (because intra zonal trips are represented by a duplicate stop time) can
+      // include the same start and end pickup times back to back and so must avoid this check.
+      const isInvalid = isFlexStopTime(col) ? false : val < previousValue
```

Comment:
```suggestion
      const isInvalid = !isFlexStopTime(col) && val < previousValue
```
You don't need a ternary here!

Reply:
👀

Reply:
See https://github.com/ibi-group/datatools-ui/pull/884/commits/8c8923d56e4cb3e91a1063ef249c71125b85ae60